Source/Components/ImageGlass.Library/Comparer/StringLogicalComparer.cs

```diff
@@ -45,44 +45,55 @@ public class StringLogicalComparer {
             if (!sp1 && sp2) return -1;
 
             int i1 = 0, i2 = 0; //current index
-            int r = 0; // temp result
-            while (true) {
+            while (true)
+            {
```

Comment:
```suggestion
            while (true) {
```

Let's keep the open bracket inline so that we can save more space.
You can run Clean menu, btw.